rust/syntect-plugin/examples/make_manifest.rs

```diff
@@ -83,9 +84,16 @@ fn main() -> Result<(), io::Error> {
 }
 
 fn lang_from_syn<'a>(src: &'a SyntaxReference) -> LanguageDefinition {
+    let mut extensions = src.file_extensions.clone();
+
+    // add support for .xiconfig
+    if extensions.contains(&String::from("toml")) {
```

Comment:
Can we not write this as `.contains("toml")`?

Reply:
I tried that. It gives an error `expected struct `std::string::String`, found str`. It is same with the `String::from("xiconfig")`.

Reply:
ah that makes sense, for some reason i was thinking it was a set instead of a vector, where this would work.

Here's an alternative solution that is faster (not checking every extension) but I don't think this is very important in a tool that is only run once in a while, during development.

```rust
fn lang_from_syn<'a>(src: &'a SyntaxReference) -> LanguageDefinition {
    let mut def = LanguageDefinition {
        name: src.name.as_str().into(),
        extensions: src.file_extensions.clone(),
        first_line_match: src.first_line_match.clone(),
        scope: src.scope.to_string(),
        default_config: None,
    };
    if def.name.as_ref() == "TOML" {
        def.extensions.push(String::from("xiconfig"));
    }
    def
}
```